cabal.project

```diff
@@ -3,3 +3,8 @@ index-state: 2022-07-01T00:00:00Z
 packages:
   cardano-prelude
   cardano-prelude-test
+
+source-repository-package
+  type: git
+  location: https://github.com/well-typed/canonical-json
+  tag: 8a37c22f0ce4759b4d534a4019a473e8225f3477
```

Comment:
You need to throw a --sha comment here for Nix

Reply:
That has been dropped in favor of a Hackage dependency.